solarforecastarbiter/io/utils.py

```diff
@@ -269,33 +268,39 @@ def adjust_timeseries_for_interval_label(data, interval_label, start, end):
     return data.loc[start:end]
 
 
-def serialize_data(values):
-    serialized_buf = pa.serialize(values).to_buffer()
-    compressed_bytes = zlib.compress(serialized_buf)
-    encoded = base64.b64encode(compressed_bytes)
-    return encoded.decode('ascii')  # bytes to str
+def serialize_timeseries(ser):
+    if not (
+            isinstance(ser, pd.Series) and
+            isinstance(ser.index, pd.DatetimeIndex) and
+            ser.index.tzinfo is not None
+    ):
+        raise TypeError(
+            'Only pandas Series with a localized DatetimeIndex is supported')
+    v = ser.copy()
+    v.index.name = 'timestamp'
+    jsonvals = v.tz_convert('UTC').reset_index(name='value').to_json(
+        orient='records', date_format='iso', date_unit='s')
+    return jsonvals
```

Comment:
Is this the best? Should some more context be included in case it needs to be extended in the future?